tsl/test/expected/dist_hypertable-12.out

```diff
@@ -3176,6 +3188,22 @@ $$);
  
 (1 row)
 
+SELECT * FROM distributed_exec($$
```

Comment:
What are we trying to test here and what would be the desired/expected outcome?

I guess we want people to generally manage this via the access node, i.e., we'd deparse and send attach/detach tablespace to data node rather than requiring distributed_exec (in which case I would say people are on their own).

Reply:
@erimatnor I don't think you can manage tablespaces through the access node. At least there are some tradeoffs, since tablespaces  are created locally on data nodes, and I guess `dist_exec` will be away. If it will be managed through the access node, then managing tablespaces on data node needs to be blocked.

The test checks that data nodes don't have logic to process the statement unexpectedly, so it will not be broken unintentionally. It's difficult to imagine, but I thought it doesn't hurt to test. I can remove it.